catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.tsx

```diff
@@ -231,177 +231,182 @@ const SchemaTable: FunctionComponent<Props> = ({
 
   return (
     <>
-      <table className="tw-w-full" data-testid="schema-table">
-        <thead>
-          <tr className="tableHead-row">
-            <th className="tableHead-cell">Column Name</th>
-            <th className="tableHead-cell">Data Type</th>
-            <th className="tableHead-cell">Description</th>
-            <th className="tableHead-cell tw-w-60">Tags</th>
-          </tr>
-        </thead>
-        <tbody className="tableBody">
-          {searchedColumns.map((column, index) => {
-            return (
-              <tr
-                className={classNames(
-                  'tableBody-row',
-                  !isEven(index + 1) ? 'odd-row' : null
-                )}
-                data-testid="column"
-                id={column.name}
-                key={index}
-                ref={rowRef}>
-                <td className="tw-relative tableBody-cell">
-                  {getConstraintIcon(column.columnConstraint)}
-                  <span>{column.name}</span>
-                </td>
+      <div className="tw-table-responsive">
```

Comment:
Why did we make schema table responsive?

Reply:
because it was going outside the viewport.